src/coreclr/inc/corinfo.h

```diff
@@ -317,6 +317,13 @@ struct SYSTEMV_AMD64_CORINFO_STRUCT_REG_PASSING_DESCRIPTOR
     }
 };
 
+// Represents vector information
+struct CORJIT_VECTOR_INFO
```

Comment:
nit: use CORINFO, not CORJIT. (CORJIT is used a couple times in corjit.h, the interface implemented by the JIT, not the interface implemented by the VM)
```suggestion
struct CORINFO_VECTOR_INFO
```

Reply:
Fixed.